seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,6 +181,27 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
+    # trigger dags
+    anvil_type = 'AnVIL_{sample_type}'.format(sample_type=request_json['sampleType'])
```

Comment:
so this isn't really an anvil "type", this is something specific to the dag although its not clear to me what. It would probably be better to rename this to something more descriptive